bounties/other/DeepSpeech/1/vulnerability.json

```diff
@@ -0,0 +1,109 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2020-11-28",
+  "AffectedVersionRange": "*",
+  "Summary": "Arbitrary Code Execution",
+  "Contributor": {
+    "Discloser": "B3EF",
+    "Fixer": ""
+  },
+  "Package": {
+    "Registry": "other",
+    "Name": "DeepSpeech",
+    "URL": "https://github.com/mozilla/DeepSpeech"
+  },
+  "CWEs": [
+    {
+      "ID": "CWE-94",
+      "Description": "Arbitrary Code Execution caused by by usage of load()"
+    }
+  ],
+  "CVSS": {
+    "Version": "3.1",
+    "AV": "",
+    "AC": "",
+    "PR": "",
+    "UI": "",
+    "S": "",
+    "C": "",
+    "I": "",
+    "A": "",
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "9.x"
```

Comment:
Too high in my opinion for an attack which requires anyway `write permissions` and `run` for a file in a specific directory:
![Screenshot from 2020-12-05 13-25-09](https://user-images.githubusercontent.com/33063403/101243044-d13d1880-36fd-11eb-9d8c-bb2ec240baae.png)